src/state/routing/usePreviewTrade.ts

```diff
@@ -78,9 +79,10 @@ export function usePreviewTrade(
     inputTax,
     outputTax,
   })
+  const isWindowVisible = useIsWindowVisible()
 
   const { isError, data: tradeResult, error, currentData } = useGetQuickRouteQueryState(queryArgs)
-  useGetQuickRouteQuery(skipFetch ? skipToken : queryArgs, {
+  useGetQuickRouteQuery(skipFetch || !isWindowVisible ? skipToken : queryArgs, {
```

Comment:
logic here is whack, `skipFetch` and `!isWindowVisible` can both be true, or `queryArgs` can be `skipToken` itself - we should probably consolidate whether it should skip or not in one place lol